corehq/apps/userreports/rebuild.py

```diff
@@ -37,8 +37,6 @@ def get_completed_case_type_or_xmlns(self):
         return self._client.lrange(self._key, 0, -1)
 
     def add_completed_case_type_or_xmlns(self, case_type_or_xmlns):
-        if case_type_or_xmlns is None:
-            case_type_or_xmlns = 'None'
```

Comment:
I think you don't need to roll this back. I think all these test changes are backwards compatible?

Reply:
I thought about that...but just didn't want to wait again for tests to fail and then make the changes. it's easy to add back in anyway